content/en/logs/log_configuration/rehydrating.md

```diff
@@ -83,6 +83,12 @@ One hour later, the historical view is definitively deleted; until that time, th
 
 {{< img src="logs/archives/log_archives_rehydrate_delete.mp4" alt="Deleting Historical Views" video="true"  width="75%" >}}
 
+### Viewing deleted historical views
+
+View deleted historical views for up to 90 days in the past using the `View` dropdown menu:
+
+{{< img src="logs/archives/log_archives_deleted_rehydrations.png" alt="Deleting Historical Views" video="true"  width="75%" >}}
```

Comment:
```suggestion
{{< img src="logs/archives/log_archives_deleted_rehydrations.png" alt="Deleting Historical Views"  width="75%" >}}
```

Reply:
I'm replacing an image at the top of the page as well, as it is outdated UI!